internal/exampleoc/enum.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 Google LLC
```

Comment:
i believe this package was intentional not-internal for some featureprofiles unit test

Reply:
Removing dependency in https://github.com/openconfig/featureprofiles/pull/1958